articles/if-you-like-it-then-you-shouldnt-put-some-code-in-it.md

```diff
@@ -0,0 +1,37 @@
+---
+authorName: Jonathan Rothwell
+authorGithubUsername: jrothwell
+issue: 94
+title: If you like it then you shouldn't put some code in it
+---
+# {{page.title}}
+
+In 2017, a crowdfunding campaign from Australia claimed that “in a world of technical overload there is sometimes no space to fit the simple things on your \[bicycle\] handlebars.” They wanted AU$85,000 to manufacture ‘the simple bike bell’.
+
+The ‘simple’ bike bell worked like this: Instead of of striking an actual clapper against a metal bell, you have a small button which you mount on your handlebars. You pair the button with your phone (which must run a supported operating system.) When you want to use the bell, you press the Bluetooth button. This sends a message to your phone (hopefully charged, within range, with Bluetooth turned on, and with the manufacturer’s app open and running.) Provided these conditions are all met, your phone’s speaker plays the sound of a bicycle bell... probably. What could be simpler than that? (Kickstarter users only pledged 1.7% of its funding target.)
+
+What are the thought processes that lead to trashy or tasteless IoT solutions like this? It’s simple: someone somewhere sees an itch, which they scratch with what they know—technology. They then pat themselves on the back for being so ‘disruptive’, make a few news releases, and voilà: profit. (Maybe.)
+
+Meanwhile, on our morning coffee break, we read the news and roar with laughter. “Remember that awful Fitbit for dogs thing? They were running a six-year-old version of Android. Now they’ve all been pwned and being used to mine bitcoin! Hah! What were they thinking?”
+
+We then walk back into the office, and proceed to make exactly the same mistakes as the people who make these terrible ‘connected’ cat litter trays and ‘smart’ toasters. As software developers, we follow the same thought process every day. We see someone with a problem, dig into our toolbox of technical solutions, and try to sell one of them to fix it in a cack-handed way.
```

Comment:
„Cack-handed“ is a new word to my English dictionary. Is this a common term?
I guess I understand the meaning though...

Reply:
In the UK it’s a common way of saying “clumsy” (you can look up what “cack” is).

We could probably replace with “clumsy” or similar.